server/app/views/applicant/ApplicantLayout.java

```diff
@@ -386,17 +386,34 @@ private DivTag authDisplaySection(
               a(createAnAccountMessage)
                   .withHref(createAnAccountLink)
                   .withClasses(ApplicantStyles.LINK)));
-    } else {
-      String loggedInAsMessage =
-          messages.at(MessageKey.USER_NAME.getKeyName(), personalInfo.getDisplayString(messages));
-      String logoutLink = org.pac4j.play.routes.LogoutController.logout().url();
-      return outsideDiv.with(
-          div(loggedInAsMessage).withClasses("text-sm"),
-          a(messages.at(MessageKey.BUTTON_LOGOUT.getKeyName()))
-              .withId("logout-button")
-              .withHref(logoutLink)
-              .withClasses(ApplicantStyles.LINK));
     }
+
+    String accountIdentifier =
```

Comment:
**suggestion (non-blocking):** Use a comment to explain why we are displaying email address for TIs rather than name.

Reply:
Done